text/0233-opentelemetry-sandbox.md

```diff
@@ -0,0 +1,89 @@
+
+> Project name:
+>
+> Repository name:
+>
+> Motivation:
+>
+> Zoom room requested?
+
+##### Example
+
+> Project name: OpenTelemetry Collector Community Distributions
+>
+> Repository name: opentelemetry-collector-distributions
+>
+> Motivation: The OpenTelemetry Collector Builder allows people to create their own distributions, and while the OpenTelemetry Collector project has no intentions (yet) on hosting other more specialized distributions, some community members are interested in providing those distributions, along with best practices on building and managing such distributions, especially around the CI/CD requirements.
+>
+> Zoom room requested? No
+
+### Periodic reports
+
+On the first [GC/TC joint call](https://docs.google.com/document/d/1jylE5uZCKV9mrPw8Qrc5ExGyRVbBdqcbWPwni-hB5dE) of the calendar quarter (Jan, Apr, Jul, Oct), the TC/GC sponsors for the sandbox projects MUST provide an update about their sponsored project. Should the sponsor not be able to join the call, a short written report MUST be provided as part of the meeting notes.
+
+### Exiting the sandbox
+
+Projects can be hosted for at most 12 months and are expected to exit before that, with one of the following outcomes:
+
+- incorporation as part of an existing SIG
+- acceptance as a new SIG
+- archival
+
+When the sandbox project cannot get accepted as its own SIG nor incorporated as part of an existing SIG, it's automatically archived after 12 months.
+
+### Further details
+
+* A new GitHub user group will be created with the current members of the TC and GC as members. This group shall be the admin for all repositories in the organization.
+* Project proponents are added as maintainers and encouraged to recruit other maintainers from the community.
+* Code hosted under this organization is owned by the OpenTelemetry project and is under the governance of OTel’s Governance Committee.
+
+## Trade-offs and mitigations
+
+None.
+
+## Prior art and alternatives
+
+* One obvious alternative is to let users collaborate on their own accounts and organizations, as some are doing today. There are two problems with that: the first is that those projects usually lack clear governance rules, so that external contributors aren't sure what's going to happen with their code contributions. The second problem is lack of visibility: most of the current experiments and initiatives aren't listed in the [OpenTelemetry's registry](https://opentelemetry.io/ecosystem/registry/), and being under an OpenTelemetry organization would make those projects more visible while still at early stages.
+* Another alternative is to direct users to CNCF's sandbox, but at this moment, the queue of projects applying for that is huge and there's no expectations that a project applying today would be considered within the next few months. Besides, the CNCF Sandbox implies that projects are standalone projects, while OpenTelemetry Sandbox projects would often be incorporated as part of an existing SIG.
```

Comment:
I like your optimism on the success of the sandbox. If we end up having such a big number of projects applying to the sandbox, I'm sure we can figure out a solution.

Reply:
How about a compromise: we commit to having at most 3 projects in the sandbox for the first year. It shouldn't have too much overhead for the TC and we can use this experience to see if this would scale.

Reply:
I added this limit of 3 projects to the proposal.